transliterate/test_transliterate.py

```diff
@@ -0,0 +1,7 @@
+from transliterate import transliterate
+
+response = transliterate("સંકેત")
```

Comment:
you can put this line inside `test_response` and make a unit test idempotent. 
If you also make the change I mentioned of accepting languages from parameters, then you'll be able to easily extend tests for different languages.